manifests/init.pp

```diff
@@ -9,6 +9,7 @@
   $init_config_hash     = {},
   $manage_init_defaults = false,
   $manage_package       = true,
+  $manage_repo          = $::syslog_ng::params::manage_repo,
```

Comment:
I am not convinced about a fixed value here, but neither could decide which platforms to enable and disable by default…  For sure, I do use a custom repo on Debian nodes, and now on CentOS ones…  I would not use it on FreeBSD but I do not use the module at home for now, so have not looking into it :wink: 

Maybe we can "think" about it for now, and fine-tune this in a migration from `params.pp` to hiera data?